config/locales/en.yml

```diff
@@ -2915,22 +2928,6 @@ en:
         unsubscribe: "Unsubscribe"
         hide_comment: "hide"
         unhide_comment: "unhide"
-    notes:
-      new:
-        intro: "Spotted a mistake or something missing? Let other mappers know so we can fix it. Move the marker to the correct position and type a note to explain the problem."
-        advice: "Your note is public and may be used to update the map, so don't enter personal information, or information from copyrighted maps or directory listings."
-        add: Add Note
-      show:
-        anonymous_warning: This note includes comments from anonymous users which should be independently verified.
-        hide: Hide
-        resolve: Resolve
-        reactivate: Reactivate
-        comment_and_resolve: Comment & Resolve
```

Comment:
Actually `comment` and `comment_and_resolve` are used from javascript in `note.js` where the button value is updated as text is entered...